static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js

```diff
@@ -114,7 +110,7 @@ define([
                 inlineMercPromise = Promise.resolve(null);
             }
 
-            if (config.switches.viewability && !inMobileAdsTest()) {
+            if (config.switches.viewability) {
```

Comment:
Nice catch. We need the mobile check for this. I've also included a few new unit tests so it should be nicely covered now.